README.md

```diff
@@ -1,23 +1,34 @@
 # The BIG List of Video Game Randomizers
 
-- New website: <https://video-game-randomizers.github.io/rando-list/>
-- Old website: <https://www.debigare.com/randomizers/>
+- New website: <https://randomizers.debigare.com>
+- New website (backup): <https://video-game-randomizers.github.io/rando-list/>
 - Discord Community Server: <https://discord.com/invite/YREMzGQ3gd>
 
 ## About
 
-This repository contains the source code for [The BIG List of Video Game Randomizers](https://video-game-randomizers.github.io/rando-list/), a community project created and originally published in 2016 by [Guillaume Fortin-Debigaré](https://www.debigare.com/). The project has been continuously maintained by him based on countless contributions by the community over the years, up until it became open source in 2024.
+This repository contains the source code for [The BIG List of Video Game Randomizers](https://randomizers.debigare.com), a community project created and [originally published](https://www.debigare.com/randomizers/) in 2016 by [Guillaume Fortin-Debigaré](https://www.debigare.com/).
 
-The project is currently hosted on [Guillaume Fortin-Debigaré's personal website](https://www.debigare.com/randomizers/), although we are currently working on moving it to <https://video-game-randomizers.github.io/rando-list/>.
+The project has been continuously maintained by him based on countless contributions by the community over the years, and is now maintained directly via open-source contributions.
 
 ## How to contribute
 
-Pull requests from the public are very welcome. If you want to become a maintainer of this project or you need help with contributing, please join the [official Discord Community Server](https://discord.com/invite/YREMzGQ3gd) and post in the `#list-maintenance` channel.
+You can request updates to the list either by submitting an issue with the `Submit New Randomizer` template, or in the `#list-submissions` in the [Discord](https://discord.com/invite/YREMzGQ3gd).
```

Comment:
could hotlink to the create issue page

```suggestion
You can request updates to the list either by submitting an issue with the [`Submit New Randomizer` template](https://github.com/video-game-randomizers/rando-list/issues/new/choose), or in the `#list-submissions` in the [Discord](https://discord.com/invite/YREMzGQ3gd).
```

or even https://github.com/video-game-randomizers/rando-list/issues/new?assignees=&labels=submission&projects=&template=2-submission.yml&title=%5BSubmission%5D%3A+ 

but the generic link might be better if people see they can report bugs and feature requests too, and also more resilient to our changes